lib/committee/drivers/open_api_2/link.rb

```diff
@@ -23,13 +23,20 @@ class Link
 
         # The link's output schema. i.e. How we validate an endpoint's response
         # data.
-        attr_accessor :target_schema
+        attr_accessor :target_schemas
 
         attr_accessor :header_schema
 
         def rel
           raise "Committee: rel not implemented for OpenAPI"
         end
+
+        def target_schema
+          target_schemas[status_success] ||
+            target_schemas[200] ||
+            target_schemas[201] ||
+            target_schemas.values.first
+        end
```

Comment:
Since I already call `find_best_fit_response` in the driver, I'm not sure this is needed here. Maybe just calling `target_schemas[status_success]` would be enough?
This is mostly for compatibility with hyper schema anyway.